File: 7-kyu/geometry-basics--triangle-perimeter-in-2d/python/solution.py
from math import sqrt

def dist(a, b):
    return sqrt((a.x - b.x) ** 2 + (a.y - b.y) ** 2)

def triangle_perimeter(triangle):
    ret = dist(triangle.a, triangle.b) + dist(triangle.b, triangle.c) + dist(triangle.a, triangle.c)
    return round(ret, 6)

File: 7-kyu/geometry-basics--triangle-perimeter-in-2d/javascript/solution.js
const trianglePerimeter = triangle => {
  const distance = (a, b) => Math.sqrt((a.x - b.x) ** 2 + (a.y - b.y) ** 2);
  const perimeter = distance(triangle.a, triangle.b) + distance(triangle.b, triangle.c) + distance(triangle.c, triangle.a);
  return +perimeter.toFixed(6);
};
